libs/portals/admin/regulations-admin/src/components/EditBasics.tsx

```diff
@@ -8,9 +8,10 @@ import {
   Button,
   AlertMessage,
   AlertBanner,
+  Select,
 } from '@island.is/island-ui/core'
 import { EditorInput } from './EditorInput'
-import { editorMsgs as msg, errorMsgs } from '../lib/messages'
+import { editorMsgs as msg, errorMsgs, m, editorMsgs } from '../lib/messages'
```

Comment:
Why import the same thing, one aliased?

Reply:
obviously there's a good reason for that!!